### PR TITLE
Alpha branding

### PR DESCRIPTION
**Pull request by teneightfive**
## User Story
As PM I'd like alpha branding on the site and tools so staff and citizens do not think it is a finished product ([#95958206](https://www.pivotaltracker.com/story/show/95958206))

**Note:** this merges into internal-template (#33).

### CODE REVIEW

File: mtp_cashbook/templates/base.html [hunk 1]
```diff
@@ -41,4 +41,13 @@
       {% endfor %}
   </ul>
   {% endif %}
+
+  <div class="subheader-item phase-banner">
+    <span class="phase-tag phase-tag-alpha">
+      ALPHA
```
Comment (matharden) [inline mtp_cashbook/templates/base.html]: Could this be wrapped in a translation filter please?

Comment (SteveMarshall) [inline mtp_cashbook/templates/base.html]: Also: lowercase text and style it to uppercase using CSS? (If memory serves, most screen readers will read this as “Ay Ell Pee Haitch Ay”?)

Comment (matharden) [inline mtp_cashbook/templates/base.html]: :+1:

File: mtp_cashbook/templates/base.html [hunk 2]
```diff
@@ -41,4 +41,13 @@
...
+    </span>
+    <span class="phase-banner-message">
+      This is a new service – your <a href="#">feedback</a> will help us to improve it.
```
Comment (SteveMarshall) [inline mtp_cashbook/templates/base.html]: Likewise, this should be `blocktrans`'d.

**Comment by teneightfive** [comment]
@matharden @SteveMarshall @marcofucci this okay to merge?

File: mtp_cashbook/templates/base.html [hunk 3]
```diff
@@ -41,4 +41,10 @@
       {% endfor %}
   </ul>
   {% endif %}
+
+  <p class="subheader-item phase-banner">
+    <strong class="phase-tag phase-tag-alpha">{% trans "Alpha" %}</strong>
+    {# #TODO - add current location for feedback view #}
+    {% blocktrans %}This is a new service – your <a href="#">feedback</a> will help us to improve it.{% endblocktrans %}
```
Comment (SteveMarshall) [inline mtp_cashbook/templates/base.html]: *koff* `#TODO` *koff*

**Comment by marcofucci** [comment]
@teneightfive you can merge it

**Pull request merged**